yarn.lock

```diff
@@ -2,3 +2,53 @@
 # yarn lockfile v1
 
 
+asynckit@^0.4.0:
```

Comment:
It appears that you have a package.json and yarn.lock in the root directory. This is probably a mistake. These files should be deleted, and anything that needs to be installed should be done in /onboard directory.

Reply:
Deleted